cms/templates/cms/home_page.html

```diff
@@ -7,6 +7,12 @@
 {% block body_class %}template-programpage{% endblock %}
 
 {% block title %}{% trans "MIT MicroMasters" %}{% endblock %}
+{% block description %}
+  The MicroMasters credential is a new option for online learning
+  in which the students can earn a full master's degree by taking
+  about half the course content online and half on campus.
```

Comment:
Here's the most recent copy from https://goo.gl/XFo3Fn

```
The MicroMasters is a new digital credential for online learners. The MicroMasters credential will be granted by MITx (MIT’s online learning initiative) to learners who excel in an integrated set of graduate-level online courses and a final capstone exam.  
```

@roberthouse54 do we have final copy yet?

If not, we should just go with this so we can move on.